src/components/Header/Header.test.js

```diff
@@ -33,5 +34,16 @@ describe("<Header />", () => {
       global.dispatchEvent(new Event('resize'));
     })
     expect(await screen.findByLabelText(/hamburger menu/i)).toBeInTheDocument();
+  });
+
+  it("the hamburger button toggles the menu", async () => {
+    act(() => {
```

Comment:
could try and put this act resize function in its own function since we're writing it twice - (and probably will use it more)

Reply:
Thanks for the feedback! Pushing this up right now. I extracted it from the tests but left it in the file (with an export), since at the moment I'm not sure where its best long-term home would be.